tests/test_steps/test_generic_steps/test_load_data_step.py

```diff
@@ -0,0 +1,103 @@
+"""Test load data step."""
+import os
+from typing import Tuple
+from unittest.mock import patch
+
+import pytest
+from pandas import DataFrame
+from pandas.testing import assert_frame_equal
+from steps.generic_steps import load_data
+
+
+def test_load_data(
+    directory_for_testing: str, sample_scraping_data: Tuple[DataFrame, DataFrame]
+):
+    """Tests that the correct functions are called when loading data.
+
+    Args:
+        directory_for_testing (str): Mock testing directory.
+        sample_scraping_data (Tuple[DataFrame, DataFrame]): Mock DataFrames for saving/loading.
+    """
+    data_location = os.path.join(directory_for_testing, "data")
+    os.mkdir(data_location)
+    mind, nhs = sample_scraping_data
+    mind.to_csv(f"{data_location}/mind_data_raw.csv", index=False)
+    nhs.to_csv(f"{data_location}/nhs_data_raw.csv", index=False)
+
+    with patch(
```

Comment:
Nit: these patches could be a fixture which yields the context back to testing function